doc/AWS.md

```diff
@@ -0,0 +1,466 @@
+
+**Fine-grained access control**
+As we are using a small instance, Fine grained access control isn't available. 
+
+**Amazon Cognito authentication**
+For this guide we will be creating an IAM role rather than using Cognito. Do not enable Amazon Cognito authentication if you are following this guide. 
+
+**Access Policy**
+To allow SAS access to the ElasticSearch cluster we need to add a custom access policy. We will have to edit this after the ElasticSearch instance has been created but for now enter set the first drop down to `IAM ARN`. In the text box enter (Enter Principal) put the User ARN you created earlier in the IAM screen. The third drop down should be `Allow`.
+
+**Encryption**
+
+Check **Require HTTPS for all traffic to the domain**. Do not select **Node-to-node encryption** or **Enable encryption of data at rest**. There is no need to fill in any of the **Optional Elasticsearch cluster settings**.
+
+Now click `Confirm` to create your ElasticSearch instance. 
+
+To aid development you may want to access ElasticSearch and Kabana from your development machine. If this is the case then select your domain and in the `Actions` drop down select **Modify access policy)**. Ensure JSON defined access policy is set and paste the following JSON:
+
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "USER_ARN"
+      },
+      "Action": "es:*",
+      "Resource": "ELASTIC_SEARCH_ARN"
+    },
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
+      },
+      "Action": "es:ESHttp*",
+      "Resource": "ELASTIC_SEARCH_ARN",
+      "Condition": {
+        "IpAddress": {
+          "aws:SourceIp": "DEV_SERVER_IP"
+        }
+      }
+    }
+  ]
+}
+```
+
+Making the following changes:
+ * **USER_ARN**: You should find this on the summary screen of the IAM user you created at the start of the Storage part of this guide.
+ * **ELASTIC_SEARCH_ARN**: You should find this on the ElasticSearch information screen for your domain. It will be named **Domain ARN**.
+ * **DEV_SERVER_IP**: To allow you to view data in ElasticSearch add your IP address so that you can view ElasticSearch and Kabana without using the IAM login. 
+
+You should now have an ElasticSearch instance that is ready to be used with SAS. 
+
+
+### Access - Elastic Load Balancer
+
+To be able to use ECS and Fargate we need to create an Elastic Load Balancer that will route incoming traffic to the correct ECS Task. To do this navigate to the EC2 service and scroll down to Load Balancers. Click Create load Balancer. 
+
+**Load balancer type**: Application Load Balancer
+
+**Name**: https-LoadBalancer
+
+**Scheme**: Internet facing
+
+**IP address type**: ipv4
+
+**Listeners**: If you want to access your application using http then leave the default listener in place. You should also add a https listener by clicking Add listener and selecting https. 
```

Comment:
If you have configured a certificate you can add a https listener but this can be added later.